src/sdk/main/src/Transaction.cc

```diff
@@ -355,8 +364,9 @@ std::vector<std::byte> Transaction<SdkRequestType>::toBytes() const
 template<typename SdkRequestType>
 SdkRequestType& Transaction<SdkRequestType>::sign(const std::shared_ptr<PrivateKey>& key)
 {
-  return signInternal(
-    key->getPublicKey(), [key](const std::vector<std::byte>& vec) { return key->sign(vec); }, key);
+  // clang-format off
+  return signInternal(key->getPublicKey(), [key](const std::vector<std::byte>& vec) { return key->sign(vec); }, key);
+  // clang-format on
```

Comment:
Is this intentional?

Reply:
Yes, this line constantly causes issues for me as I'm using the same clang-format version configured in the CI but the file isn't passing the clang-format check. I always have to manually edit this so it passes... Was thinking of why just not save the effort. If other issues with formatting appear I would investigate and try reworking my local setup. But I guess this doesn't harm the codebase that much.